hbase-client/src/main/java/org/apache/hadoop/hbase/client/SnapshotDescription.java

```diff
@@ -139,15 +146,27 @@ public long getCreationTime() {
     return this.creationTime;
   }
 
+  // get snapshot ttl in sec
+  public long getTtl() {
```

Comment:
Getters and setters for ttl is fine even if also using generic attribute map with getters and setters for elements of same.